### PR TITLE
Add Japanese language

### PR DESCRIPTION
**Pull request by mtujikawa**

### CODE REVIEW

File: youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po [hunk 1]
```diff
@@ -0,0 +1,588 @@
...
+msgid "Youtube-dl download failed [{0}]"
+msgstr "Youtube-dlのダウンロードに失敗しました [{0}]"
+
+#: youtube_dl_gui/mainframe.py:132
+msgid "Successfully downloaded youtube-dl"
+msgstr "Youtube-dlのダウンロードに成功しました"
+
+#: youtube_dl_gui/mainframe.py:134
+msgid "Unable to open directory: '{dir}'. The specified path does not exist"
+msgstr ""
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po]: Please keep those in a single line

File: youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po [hunk 2]
```diff
@@ -0,0 +1,588 @@
...
+"ディレクトリを開くことができません: '{dir}'。指定されたパスが存在しません。"
+
+#: youtube_dl_gui/mainframe.py:136
+msgid "Error while shutting down. Make sure you typed the correct password"
+msgstr ""
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po]: Same here

File: youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po [hunk 3]
```diff
@@ -0,0 +1,588 @@
+# Youtube-dlG localization file.
+# FIRST AUTHOR: Sotiris Papadopoulos <ytubedlg@gmail.com>, 2015.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: youtube-dlg 0.4\n"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po]: Why did you remove some headers like:
* POT-Creation-Date
* PO-Revision-Date
* Last-Translator

We need those for maintenance reasons. Please add them back.

**Comment by mtujikawa** [comment]
I fixed the points pointed out.
Please review again.

**Comment by MrS0m30n3** [comment]
You need to push your changes.

**Comment by mtujikawa** [comment]
Thank you for your check.
I pushed my change.
I am sorry, please review again.

File: youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po [hunk 4]
```diff
@@ -0,0 +1,588 @@
...
+"POT-Creation-Date: 2017-06-15 17:13+EEST\n"
+"PO-Revision-Date: 2017-06-15 17:14+EEST\n"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po]: Please update the revision date.

File: youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po [hunk 5]
```diff
@@ -0,0 +1,588 @@
...
+msgid "Item is not completed"
+msgstr "未完了のアイテムです"
+
+#: youtube_dl_gui/mainframe.py:668
+msgid "Update in progress. Please wait for the update to complete"
+msgstr "アップデート中です。完了までお待ちください"
+
+#: youtube_dl_gui/mainframe.py:716
+msgid "Logging is disabled"
+msgstr "ロギング無効"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po]: According to Google the msgstr field translates to "Invalid logging".
Are you sure this is correct?

File: youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po (hunk 4 above)
Comment (mtujikawa) [reply youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po]: Fixed Datetime.

File: youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po (hunk 5 above)
Comment (mtujikawa) [reply youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po]: It's OK in Japanese.

**Comment by mtujikawa** [comment]
Thank you for your check.
I am sorry, please review again.

File: youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po [hunk 6]
```diff
@@ -4,8 +4,8 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: youtube-dlg 0.4\n"
-"POT-Creation-Date: 2017-06-15 17:13+EEST\n"
-"PO-Revision-Date: 2017-06-15 17:14+EEST\n"
+"POT-Creation-Date: 2017-09-26 00:44+JST\n"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po]: You should change only the **PO-Revision-Date** and NOT the **POT-Creation-Date** (that's the date the original translation file was created). Please set the **POT-Creation-Date** back to the old value *2017-06-15 17:13+EEST*.

Everything else is fine.

**Comment by mtujikawa** [comment]
Thank you for your check.
I fixed a mistake.

**Comment by MrS0m30n3** [comment]
Everything seems fine. I will merge your pull-request as soon as i'm done with the other translations.

Thank you very much for your contribution.

**Comment by mtujikawa** [comment]
Thank you for your check!!
